src/transverse_mercator.jl

```diff
@@ -744,12 +744,12 @@ function transverse_mercator_reverse(lon0, x, y, k0, tm::TransverseMercator{MaxP
     r = hypot(s, c)
 
     if (r != 0)
-        lon = atan(s, c) * 180/pi # Krueger p 17 (25)
+        lon = atand(s, c)# Krueger p 17 (25)
 
         # Use Newton's method to solve for tau
         sxip = sin(xip)
         tau = tauf(sxip/r, tm.e2) # TODO maybe change to C++ es version
-        gamma += atan(sxip * tanh(etap), c) * 180/pi # Krueger p 19 (31)
+        gamma += atand(sxip * tanh(etap), c) * 180/pi # Krueger p 19 (31)
```

Comment:
I think the multiplication needs to be removed here as well, hopefully this will also make the tests pass

Reply:
Okay done! Thankyou!